index.js

```diff
@@ -68,24 +69,45 @@ function processRecord(record, callback) {
             var targetKey = srcKey;
             
             console.log("Copying '" + srcKey + "' from '" + srcBucket + "' to '" + targetBucketName + "'");
+            console.log(record);
             
+            console.log(eventName); 
+            if (eventName == "ObjectRemoved:DeleteMarkerCreated") {
```

Comment:
What about non-versioned S3 buckets? Is this same event triggered or is it different?